### PR TITLE
feat/#51 review and update the chat view

### LINKED ISSUE #51: Review And Update The Chat View
### Explanation:
Chat view header and footer styles will be changed, a back button will be placed, icons and colors will be changed in chat. 

### SCREENSHOTS
Chat view

 
### CHECKLIST 

- [ ] Change the colors of the `#9DBDFD` header.
- [ ] Change user icon.
- [ ] Change text styles, username in header with bold popins font and text color `#003B62`
- [ ] Place back button according to the design image.
- [ ] Change ´delivery chat´ icon.
- [ ] Change the color of the footer `#9DBDFD`
- [ ] Change color of text field `#3B3B3C`
- [ ] Replace send icon.
### REFERENCES
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=407%3A46
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=522%3A28

### PR DESCRIPTION
**Pull request by arodriguezl9302**
#### ATTENTION

PLEASE NOTE THAT TO AVOID PROBLEMS RUNNING THIS PR, YOU NEED TO HAVE PR#50 RUNNING

### Description
I have changed the chat view, the header and footer styles were changed, the back button was placed, icons and colors were changed in chat.

fixes #51 

#### Screenshots

image

#### Type of change

New feature (non-breaking change which adds functionality)

### How to test it

1. Go to '/chat/:id'

### Checklist:

- [x] Change header colors #9DBDFD
- [x] Change user icon
- [x] Change username text styles in header with bold popins font and text color #003B62
- [x] Place back button according to the design image
- [x] Change chat delivery icon
- [x] Change footer color #9DBDFD
- [x] Change text field color #3B3B3C
- [x] Replace send icon

### CODE REVIEW

File: src/components/Input.jsx
Comment (heliomar-pena) [inline src/components/Input.jsx]: They're comments? Please remove comments.

File: src/pages/Chat/Chat.styled.jsx
Comment (heliomar-pena) [inline src/pages/Chat/Chat.styled.jsx]: Please remove comments

**Comment by angelik0828** [comment]
I have reviewed, the back button does not have any color border, only the image, its functionality was affected, it does not go back but refreshes the page, on the other hand, in the tablet view, both the image and the name that appear on it upper part must be aligned to the center as in the design, previously there was only the text aligned to the center, both must be aligned, the icon used to send was not changed, in resource it is in the figma link it has a different color from the previous one, the rest of the styles and text sizes that were in the layout on which the task is based were ignored, the most important ones were listed in the list but you have to verify that the styles of text sizes and margins and spaces are already they are that this is also part of the styles.
Vista chat

image

since the icon is not centered with the text, this gives problems at smaller resolutions

The resource or image must be consistent with the resolution.

the name text is also not bold

**Comment by heliomar-pena** [comment]
I will wait until the #70 pr was merged to check, to avoid conflicts

File: src/pages/Chat/Chat.styled.jsx
Comment (heliomar-pena) [inline src/pages/Chat/Chat.styled.jsx]: What does this line? I check and removing it the header is centered

imagen

But adding it the header is not centered

imagen

I think it's bad to abuse of margin

File: src/pages/Chat/index.jsx
Comment (heliomar-pena) [inline src/pages/Chat/index.jsx:69]: I'm receiving this JSDoc error because bgColor is missed.
imagen

I would suggest to use bgColor="white" instead of delete it, and left `// ts-ignore` flag

`
              // @ts-ignore
              bgColor="white"
`

File: src/pages/Chat/index.jsx
Comment (heliomar-pena) [inline src/pages/Chat/index.jsx:85]: Please left ts-ignore flag, since without it I'm receiving JSDoc error

imagen

**Review by heliomar-pena — APPROVED** [review]
Good work! I was left you little comments

File: src/pages/Chat/Chat.styled.jsx
Comment (arodriguezl9302) [reply src/pages/Chat/Chat.styled.jsx]: this line makes the image and text next to the back button

**Comment by arodriguezl9302** [comment]
@Katsu08 @angelik0828 

I just took the suggestions and ask for this much better

remove all unnecessary comments

and the application turned out to look like this in the end
image

**Comment by angelik0828** [comment]
I see it very well I improve a lot good job I think the only thing missing is a margin in the chat field with reference to the left, we almost made it!!!

**Comment by arodriguezl9302** [comment]
@angelik0828 I just repaired this part you suggested

File: src/pages/Chat/Chat.styled.jsx
Comment (heliomar-pena) [reply src/pages/Chat/Chat.styled.jsx]: To make the image and text next to the back button you should reduce the width of the backbutton. 

How you notice, the width is 33vw
imagen

You can to use one media query to change this width in small screens
imagen

If you remove the width to 0, then the text and image will be next to the back button

File: src/pages/Chat/index.jsx
Comment (arodriguezl9302) [reply src/pages/Chat/index.jsx:85]: not having any reference on this, I decided to remove it

File: src/pages/Chat/index.jsx
Comment (arodriguezl9302) [reply src/pages/Chat/index.jsx:69]: ready

**Pull request merged**
